src/commands/init.rs

```diff
@@ -10,9 +12,13 @@ macro_rules! init_help_body {
         r#""
 error: {}
 
-An example cookbook init invocation looks like:
+An example tome init invocation looks like:
+
+# Bash/Zsh
+source <(tome init sc ~/my_script_dir $0)
 
-source <(./cookbook init sc ~/my_script_dir $0)
+# Fish
+tome init sc ~/my_script_dir $0 | source
```

Comment:
Meta: I'd like one place we define these and then re-use in various locations. As it stands now I recall 2 places these exist. One in `.rs` and one in `.md`